packages/pluggableWidgets/bar-chart-native/src/BarChart.editorConfig.ts

```diff
@@ -32,8 +32,10 @@ export function check(values: BarChartPreviewProps): Problem[] {
 
     values.barSeries.forEach((series, index) => {
         if (series.dataSet === "static") {
-            // @ts-ignore
-            if (series.staticDataSource.type === "null") {
+            if (
+                !series.staticDataSource ||
+                ("type" in series.staticDataSource && series.staticDataSource.type === "null")
```

Comment:
`("type" in lines.staticDataSource && lines.staticDataSource.type === "null")`  is equivalent to `(lines.staticDataSource.type === "null")`. Same for the other widgets :)